bl/evaluation/printer.go

```diff
@@ -171,7 +171,7 @@ func parseInvalidYamlFilesToWarnings(invalidYamlFiles []*extractor.InvalidFile)
 
 	for _, invalidFile := range invalidYamlFiles {
 		warnings = append(warnings, printer.Warning{
-			Title:       fmt.Sprintf(">>  File: %s\n", invalidFile.Path),
+			Title:       invalidFile.Path,
```

Comment:
can you explain this change?

Reply:
Sure, every warning printed the filename with this prefix so I just added it to the print function itself,
take a look at `PrintFilename` function